lib/pbench/agent/tool_meister.py

```diff
@@ -1579,9 +1575,9 @@ def end_tools(self, data: dict) -> int:
         failures = 0
         tool_cnt = 0
         for name, persistent_tool in self._persistent_tools.items():
-            assert name in self._tools, (
+            assert name in self._usable_tools, (
                 f"Logic bomb!  Persistent tool, '{name}' not in registered"
```

Comment:
Two small comments:
- I would assert (:wink:) that we don't need to use the phrase "Logic bomb" in an _assertion_ -- the semantics of `AssertionError` imply that.  🙂
- I'm seeing the phrase "Logic bomb" [in the press](https://red.ht/3upLp59) currently, referring to malware...so we might want to avoid using it in our code.  😬

Reply:
Tracked in https://github.com/distributed-system-analysis/pbench/issues/2714.